charts/invenio/templates/secret.yaml

```diff
@@ -10,12 +10,16 @@ metadata:
   annotations:
     helm.sh/resource-policy: keep
 data:
-  INVENIO_SECRET_KEY: {{ randAlphaNum 128 | b64enc | quote }}
-  INVENIO_SECURITY_LOGIN_SALT: {{ randAlphaNum 128 | b64enc | quote }}
-  INVENIO_SECURITY_PASSWORD_SALT: {{ randAlphaNum 128 | b64enc | quote }}
-  INVENIO_SECURITY_CONFIRM_SALT: {{ randAlphaNum 128 | b64enc | quote }}
-  INVENIO_SECURITY_RESET_SALT: {{ randAlphaNum 128 | b64enc | quote }}
-  INVENIO_SECURITY_CHANGE_SALT: {{ randAlphaNum 128 | b64enc | quote }}
-  INVENIO_SECURITY_REMEMBER_SALT: {{ randAlphaNum 128 | b64enc | quote }}
-  INVENIO_CSRF_SECRET_SALT: {{ randAlphaNum 128 | b64enc | quote }}
+  # retrieve the secret data to avoid creating one on every deployment
+  {{- $secretObj := (lookup "v1" "Secret" .Release.Namespace ( include "invenio.secretName" . )) | default dict }}
+  {{- $secretData := (get $secretObj "data") | default dict }}
+
+  INVENIO_SECRET_KEY: {{ default (randAlphaNum 128 | b64enc ) (get $secretData "INVENIO_SECRET_KEY") | quote }}
+  INVENIO_SECURITY_LOGIN_SALT: {{ default (randAlphaNum 128 | b64enc ) (get $secretData "INVENIO_SECURITY_LOGIN_SALT") | quote }}
+  INVENIO_SECURITY_PASSWORD_SALT: {{ default (randAlphaNum 128 | b64enc ) (get $secretData "INVENIO_SECURITY_PASSWORD_SALT") | quote }}
+  INVENIO_SECURITY_CONFIRM_SALT: {{ default (randAlphaNum 128 | b64enc ) (get $secretData "INVENIO_SECURITY_CONFIRM_SALT") | quote }}
+  INVENIO_SECURITY_RESET_SALT: {{ default (randAlphaNum 128 | b64enc ) (get $secretData "INVENIO_SECURITY_RESET_SALT") | quote }}
+  INVENIO_SECURITY_CHANGE_SALT: {{ default (randAlphaNum 128 | b64enc ) (get $secretData "INVENIO_SECURITY_CHANGE_SALT") | quote }}
+  INVENIO_SECURITY_REMEMBER_SALT: {{ default (randAlphaNum 129 | b64enc ) (get $secretData "INVENIO_SECURITY_REMEMBER_SALT") | quote }}
```

Comment:
Should `randAlphaNum 129` be `randAlphaNum 128` for consistency with the other salts?